enhancements/crane-2.1/buildconfig-to-shipwright/README.md

```
@@ -0,0 +1,378 @@
+
+#### Plugin System Prerequisite
+
+This plugin requires the crane plugin API extension that enables generating new resources (adding `NewResources []unstructured.Unstructured` to `PluginResponse`). This extension is 100% backward compatible via the `omitempty` JSON tag and is detailed in a [separate plan](https://github.com/aufi/move-crane/blob/main/drafts/plugin-update-new-resource-plan.md). Implementation touches:
+
+- **crane-lib:** `transform/plugin.go` (PluginResponse), `transform/runner.go` (RunnerResponse, Runner.Run)
+- **crane:** `Orchestrator` (artifact creation for new resources) and potentialy `Writer` (plugin dirs update).
+
+
+#### Integration with Crane Workflow
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)